src/test/java/edu/harvard/iq/dataverse/api/UtilIT.java

```diff
@@ -2485,11 +2499,15 @@ static boolean sleepForReindex(String idOrPersistentId, String apiToken, int dur
         int repeats = durationInSeconds * (1000 / sleepStep);
         boolean stale=true;
         do {
-            timestampResponse = UtilIT.getDatasetTimestamps(idOrPersistentId, apiToken);
-            System.out.println(timestampResponse.body().asString());
-            String hasStaleIndex = timestampResponse.body().jsonPath().getString("data.hasStaleIndex");
-            System.out.println(hasStaleIndex);
-            stale = Boolean.parseBoolean(hasStaleIndex);
+            try {
+                timestampResponse = UtilIT.getDatasetTimestamps(idOrPersistentId, apiToken);
+                System.out.println(timestampResponse.body().asString());
+                String hasStaleIndex = timestampResponse.body().jsonPath().getString("data.hasStaleIndex");
+                System.out.println(hasStaleIndex);
+                stale = Boolean.parseBoolean(hasStaleIndex);
+            } catch (Exception e) {
```

Comment:
It was catching illegal argument exception since apiToken can be null. But later I have fixed the UtilIT.getDatasetTimestamps such that it can take null as apiToken, so it is no longer needed. I have removed the try/catch wrapping.